rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+edition = "2018"
+fn_args_layout = "Vertical"
```

Comment:
This option is marked as deprecated in `master` branch for `rustfmt` and is replaced by `fn_params_layout`.

We talked yesterday about changing it to `Compressed`, but I've thought about it again and probably it would be better to just keep it as it is by default (`Tall`). With this option a short list of function parameters or arguments is in one line and a long list of function arguments or parameters is splitted into multiple lines, one argument or parameter in each line.

So we should just remove this line from the `rustfmt.toml` file.

Reply:
Done